src/command/dev/next/mod.rs

```diff
@@ -55,6 +55,52 @@ impl Dev {
         let (_events, subtask) = Subtask::new(router_config_watcher);
         let _abort_handle = subtask.run();
 
+        /*
+
+          something like the below; need to find where to get the right opts
+
+
+
+
+                let supergraph_yaml = self
+                    .opts
+                    .clone()
+                    .supergraph_config_source()
+                    .clone()
+                    .supergraph_yaml;
+
+                let supergraph_root = supergraph_yaml.clone().and_then(|file| match file {
+                    FileDescriptorType::File(file) => {
+                        let mut current_dir = current_dir().expect("Unable to get current directory path");
+
+                        current_dir.push(file);
+                        let path = Utf8PathBuf::from_path_buf(current_dir).unwrap();
+                        let parent = path.parent().unwrap().to_path_buf();
+                        Some(parent)
+                    }
+                    FileDescriptorType::Stdin => None,
+                });
+
+                let federation_version = self.opts.federation_version.clone();
+                let profile = self.opts.plugin_opts.profile.clone();
+                let graph_ref = self.opts.supergraph_config_source.graph_ref.clone();
+
+                let one_off_composition = OneOff::builder()
+                    .client_config(client_config)
+                    .profile(profile)
+                    .elv2_license_accepter(self.opts.plugin_opts.elv2_license_accepter)
+                    .skip_update(self.opts.plugin_opts.skip_update)
+                    .and_federation_version(federation_version)
+                    .and_graph_ref(graph_ref)
+                    .and_supergraph_yaml(supergraph_yaml)
+                    .and_supergraph_root(supergraph_root)
+                    .and_override_install_path(override_install_path)
+                    .and_output_file(output_file)
+                    .build();
```

Comment:
bunch of arguments, but this will get consolidated and cleaned up; here for purposes of demonstrating the idea